logsmanagement/README.md

```diff
@@ -0,0 +1,676 @@
+
+```
+[Docker Events Logs]
+        ## Example: Log collector that will monitor the Docker daemon socket and
+        ## collect Docker event logs in a default format similar to executing 
+        ## the `sudo docker events` command.
+
+        ## Required settings
+        enabled = yes
+        log type = flb_docker_events
+
+        ## Optional settings, common to all log source. 
+        ## Uncomment to override global equivalents in netdata.conf.
+        # update every = 1
+        # update timeout = 10
+        # use log timestamp = auto
+        # circular buffer max size MiB = 64
+        # circular buffer drop logs if full = no
+        # compression acceleration = 1
+        # db mode = none
+        # circular buffer flush to db = 6
+        # disk space limit MiB = 500
+
+        ## Use default Docker socket UNIX path: /var/run/docker.sock
+        log path = auto
+
+        ## Charts to enable
+        # collected logs total chart enable = no
+        # collected logs rate chart enable = yes
+        event type chart = yes
+        event action chart = yes
+
+        ## Stream to https://cloud.openobserve.ai/
+        output 1 name             = http
+        output 1 URI              = YOUR_API_URI
+        output 1 Host             = api.openobserve.ai
+        output 1 Port             = 443
+        output 1 tls              = On
+        output 1 Format           = json
+        output 1 Json_date_key    = _timestamp
+        output 1 Json_date_format = iso8601
+        output 1 HTTP_User        = test@netdata.cloud
+        output 1 HTTP_Passwd      = YOUR_OPENOBSERVE_PASSWORD
+        output 1 compress         = gzip
+
+        ## Real-time export to /tmp/docker_event_logs.csv
+        output 2 name             = file
+        output 2 Path             = /tmp
+        output 2 File             = docker_event_logs.csv
+```
+
+</a>
+
+<a name="troubleshooting"/>
+
+## Troubleshooting
+
+</a>
+
+1. I am building Netdata from source or a Git checkout but the `FLB_SYSTEMD` plugin is not  available / does not work: 
+
+If during the Fluent Bit build step you are seeing the following message: 
+```
+-- Could NOT find Journald (missing: JOURNALD_LIBRARY JOURNALD_INCLUDE_DIR)
+``` 
+it means that the systemd development libraries are missing from your system. Please see [how to install them alongside other required packages](https://github.com/netdata/netdata/blob/master/packaging/installer/methods/manual.md).
+
+2. Logs management and kernel log collection do not work at all and I am seeing the following error in `collector.log`:
+
+```
+[2020/10/20 10:39:06] [error] [plugins/in_kmsg/in_kmsg.c:291 errno=1] Operation not permitted
+[2020/10/20 10:39:06] [error] Failed initialize input kmsg.0
+```
+Netdata is executed without root permissions, so the kernel ring buffer logs may not be accessible from normal users. Please try executing:
```

Comment:
@netdata/agent-sre I consider this suggestion a little bit complicated, can you confirm we should remove it?